services/provider/server/storageclaim.go

```diff
@@ -64,6 +64,10 @@ func (s *storageClassClaimManager) Create(ctx context.Context, consumer *ocsv1al
 			Name:            generatedClaimName,
 			Namespace:       s.namespace,
 			OwnerReferences: []metav1.OwnerReference{ownerRef},
+			Annotations: map[string]string{
+				consumerUUIDAnnotation:          consumerUUID,
```

Comment:
OwnerRef already has consumer UUID. Do we need it as annotation also?
Annotations can't be used as label selector when doing get or list operations. If it's for tracking, label would be a better choice.

Reply:
Yes it's for tracking purposes and also helps with listing all claims of a consumer also, let me move it to labels.